### PR TITLE
Add parameters for rerun in `start_launch`

### PR DESCRIPTION
**Pull request by doyou89**
ReportPortal now supports a functionality through the API which makes rerunning tests possible. This makes the client support it.

Covers issues #107 and #108.

### CODE REVIEW

**Comment by iivanou** [comment]
It does not address any of the mentioned issues. Also, updating a dictionary with the specific key names is not safe. Unit tests are missing as well.

**Comment by doyou89** [comment]
I understand your opinion. Then I'll remove updating `kwargs` and just add parameters for `rerun`. And unit test also.

**Review by Scanters — APPROVED** [review]

File: reportportal_client/service.py [hunk 1]
```diff
@@ -214,6 +214,9 @@ def start_launch(self,
                      description=None,
                      attributes=None,
                      mode=None,
+                     uuid=None,
```
Comment (iivanou) [inline reportportal_client/service.py:217]: What is uuid? UUID of the launch that is being rerun should be sent as rerunOf value. [Documentation](https://github.com/reportportal/documentation/blob/master/src/md/src/DevGuides/reporting.md#start-launch)

Comment (iivanou) [reply reportportal_client/service.py:217]: Because you should use `rerun=True + rerunOf=uuid`. UUID is auto-generated.

Comment (doyou89) [reply reportportal_client/service.py]: You are right. "uuid" parameter is removed.

File: reportportal_client/service.py [hunk 2]
```diff
@@ -214,6 +214,8 @@ def start_launch(self,
                      description=None,
                      attributes=None,
                      mode=None,
+                     rerun=None,
```
Comment (iivanou) [inline reportportal_client/service.py:217]: Should be `False` by default.

**Review by iivanou — COMMENTED** [review]
One more change and we merge this.

File: reportportal_client/service.py (hunk 2 above)
Comment (doyou89) [reply reportportal_client/service.py:217]: Ok. I'll change this.

**Review by iivanou — APPROVED** [review]

**Pull request merged**